src/app/tab-blocks/tab-blocks.page.ts

```diff
@@ -16,18 +17,16 @@ import { ValidatorUtils } from '../utils/ValidatorUtils'
 export class TabBlocksPage implements OnInit {
 	public classReference = UnitconvService
 
+	dataSource: InfiniteScrollDataSource<BlockResponse>
```

Comment:
Having many blocks on the block page and scrolling down for a bit causes the following error:
```
Error while writing to local storage (might be full): DOMException: Failed to execute 'setItem' on 'Storage': Setting the value of 'CapacitorStorage.cachemodule2_api' exceeded the quota.
```

Reply:
Addressed by changing how we cache api requests in localStorage